profile.html

```diff
@@ -45,9 +45,9 @@
 
 <body>
 <div class='container'>
-  <div class='content-grid mdl-grid'>
-  <div class='mdl-cell mdl-cell--12-col mdl-cell--4-col-phone'>
-    <img class='background-photo' id="background-photo" src="https://static.wixstatic.com/media/de271e_197b08d22f27418eb9d45e9c52b135b8~mv2.jpg/v1/fill/w_790,h_243,al_c,lg_1,q_80/de271e_197b08d22f27418eb9d45e9c52b135b8~mv2.webp">
+  <div class='content-grid mdl-grid'> 
+  <div class='mdl-cell mdl-cell--12-col mdl-cell--4-col-phone' id ="background-div" >
+    <img class='background-photo' id="background-photo" src="/images/DefaultDoWhop_banner.jpg" >
```

Comment:
When you refactor upload the image to  /app-image-assets on Firebase storage and then link to it there.